test/util.js

```diff
@@ -0,0 +1,108 @@
+// Load local .env file config if it exists
```

Comment:
I think we need a general 'how to record new fixtures/how the whole nockback thing works' comment bit somwhere.  Maybe here?  Maybe in the README?